docs/4.4/preview/teleport-application-access.md

```diff
@@ -0,0 +1,207 @@
+|-|-|
+| `auth_servers` | Address of Auth or Proxy Service setup above |
+| `auth_token` | Token used to connect to other Teleport processes  |
+
+```yaml
+teleport:
+  # nodename of machine running process.
+  nodename: wellington
+  # The Auth token `tctl tokens add --type=app` or the static token. 4c7e15
+  auth_token: "4c7e15"
+  # This is the location of the Teleport Auth Server or Public Proxy
+  auth_servers:
+    - teleport.example.com:3080
+auth_service:
+  enabled: no
+proxy_service:
+  enabled: no
+ssh_service:
+   enabled: no
+# The app_service is new service. Used to access web applications.
+app_service:
+   enabled: yes
+   apps:
+   - name: "internal-dashboard"
+     # URI and Port of Application. If Teleport is installed on the host
+     # this could be a loopback address
+     uri: "http://10.0.1.27:8000"
+     # This version requires a public_addr for all Apps, these
+     #  applications should have a certificate and DNS setup
+     public_addr: "internal-dashboard.acme.com"
+     # Optional Labels
+     labels:
+        name: "acme-dashboard"
+     # Optional Dynamic Labels
+     commands:
+     - name: "os"
+       command: ["/usr/bin/uname"]
+       period: "5s"
+   - name: "arris"
+     uri: "http://localhost:3001"
+     public_addr: "arris.example.com"
+   # Teleport Application Access can be used to proxy any HTTP endpoint
+   # Note: Name can't include any spaces
+   - name: "hackernews"
+     uri: "https://news.ycombinator.com"
+     public_addr: "hn.example.com
+```
+
+#### Start Teleport
+
+```
+$ teleport start -d --config=/etc/teleport.yaml
+```
+
+#### Update DNS
+
+In the above config example, I've configured a range of applications with `public_addr`. Each of these needs
+to be set up in DNS using an `A`, `CNAME` or `AAAA` record pointing to the IP of the Teleport Proxy server.
+
+For the beta, we would recommend using a wildcard cert for TLS. You can also use an individual certificate on the Teleport Main process using the new `https_keypairs` option.
+
+```yaml
+proxy_service:
+  #... Example Snippet for new https_keypairs.
+  https_keypairs:
+  - key_file: /etc/letsencrypt/live/jwt.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/jwt.example.com/fullchain.pem
+  - key_file: /etc/letsencrypt/live/hn.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/hn.example.com/fullchain.pem
+  - key_file: /etc/certs/privkey.pem
+  - cert_file: /etc/certs/fullchain.pem
+```
+
+## View Applications in Teleport
+
+`https://[cluster-url]:3080/web/cluster/[cluster-name]/apps`
+
+<!--<video autoplay loop muted playsinline style="width:100%">
+  <source src="/video/app.mp4" type="video/mp4">
+  <source src="/video/app.webm" type="video/webm">
+Your browser does not support the video tag.
+</video>-->
+
+## Product Feedback
+
+We really value early feedback for our products. [You can easily pick a time here to have a chat on your feedback.](https://calendly.com/c/AEBVTDEDCQ2Z2YSB?month=2020-10)
```

Comment:
Is this link still legit? (it's for last month)

Reply:
```suggestion
We really value early feedback for our products. [You can easily pick a time here to have a chat on your feedback.](https://calendly.com/c/AEBVTDEDCQ2Z2YSB?month=2020-11)
```